### PR TITLE
443-remove pronoun and custom_pronoun references and columns from members

### PR DESCRIPTION
**Pull request by italomatos**
# What it does

Remove pronoun and custom_pronoun from member table, and remove all reference about these columns.

Issue link: https://github.com/rubyforgood/circulate/issues/443

### CODE REVIEW

File: app/models/member.rb [hunk 1]
```diff
@@ -20,14 +20,13 @@ class Member < ApplicationRecord
   has_many :notes, as: :notable
 
   PRONOUNS = ["he/him", "she/her", "they/them"]
-  enum pronoun: [:"he/him", :"she/her", :"they/them", :custom_pronoun]
+  enum pronoun: [:"he/him", :"she/her", :"they/them"]
```
Comment (jim) [inline app/models/member.rb:23]: I think we can remove this line entirely since we're removing the `pronoun` column.

**Review by jim — CHANGES_REQUESTED** [review]
This looks good! There is one modified line that can be removed entirely. Thanks for the contribution.

File: app/models/member.rb (hunk 1 above)
Comment (italomatos) [reply app/models/member.rb:23]: Make sense 👍🏻

**Review by jim — APPROVED** [review]
This is great! Thanks.

**Pull request merged**